### DISCUSSION TITLE
APIs being excessively spammed

### DISCUSSION THREAD

**Discussion by zoddshop** [rhubarbgroup/.github#35] [Support (ROC)] [answered]
Another well known plugin author has told me this

Your one of the plugins  "Redis Object Cache"   causing this issue,it  prevents the all wordpress transient from  creation that causes excessive api calls not only our plugin api instead all wordpress api's.
for now i have deactivated it on your site so you can check.

**Comment by tillkruss** [comment, marked as answer]
WordPress core will use `wp_cache_*()` functions for all Transients when _Redis Object Cache_ is enabled. This works fine, if you API "is being spammed" than a 3rd party plugin is not compatible with WordPress' persistent object cache API.
